src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminAuditEndpointTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.admin.Audit;
+import uk.gov.hmcts.dts.fact.model.admin.OpeningTime;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.*;
+
+@ExtendWith(SpringExtension.class)
+public class AdminAuditEndpointTest extends AdminFunctionalTestBase {
+
+    private static final String ADMIN_AUDIT_ENDPOINT = "/admin/audit/";
+
+    private static final String OPENING_TIMES_PATH = "/openingTimes";
+    private static final String ADMINISTRATIVE_COURT_SLUG = "administrative-court";
+    private static final String ADMINISTRATIVE_COURT_OPENING_TIMES_PATH = ADMIN_COURTS_ENDPOINT + ADMINISTRATIVE_COURT_SLUG + OPENING_TIMES_PATH;
+
+    private static final String TEST_HOURS = "test hour";
+    private static final int BAILIFF_OFFICE_OPEN_TYPE_ID = 5;
+    private static final OpeningTime TEST_OPENING_TIME = new OpeningTime(BAILIFF_OFFICE_OPEN_TYPE_ID, TEST_HOURS);
+    private static final String TEST_AUDIT_NAME = "Update court opening times";
+
+    /************************************************************* Get Request Tests. ****************************************************/
+
+    @Test
+    public void shouldReturnAllAuditsForPageAndSize() throws JsonProcessingException {
+        setUpOpeningTimes();
+        checkAuditData("", "", "", "");
```

Comment:
We could create overloads of the checkAuditData method and supply the default values. Maybe over the top for a test class, though.

Why doesn't java allow default parameter values to be set...?